web/components/templates/requestsV2/requestRow.tsx

```diff
@@ -151,6 +158,57 @@ const RequestRow = (props: {
     }
   };
 
+  const onAddScoreHandler = async (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    setIsScoresAdding(true);
+
+    const formData = new FormData(e.currentTarget);
+    const key = formData.get("key") as string;
+    const value = formData.get("value") as any as number;
+
+    if (currentScores && currentScores[key]) {
+      setNotification("Score already exists", "error");
+      setIsScoresAdding(false);
```

Comment:
nit: in the future, to avoid having to set states everything for loading, use useMutation

https://tanstack.com/query/v4/docs/framework/react/reference/useMutation